src/BasicRum/Diagram/Builder/RenderType/RenderTypeFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\BasicRum\Diagram\Builder\RenderType;
+
+use App\BasicRum\DiagramOrchestrator;
+use App\BasicRum\Release;
+
+class RenderTypeFactory
+{
+    private $renderType;
+
+    private $renderTypeClassMap = [
+        'plane' => Plane::class,
+        'time_series' => TimeSeries::class,
+        'distribution' => Distribution::class,
+    ];
+
+    public function __construct(string $type)
+    {
+        $this->renderType = $type;
```

Comment:
It will be great if we throw an exception in case the render type doesn't exist in $this->renderTypeClassMap

Reply:
Added new exception if no suitable object factory found